### ISSUE TITLE
Meteor 1.4: Hangs on minifying app stylesheet

### ISSUE THREAD

**Issue by joncursi** [Meteor-Community-Packages/meteor-postcss#22]
The new Meteor 1.4 hangs on startup on `Minifying app stylesheet` when this package is installed:

![screen shot 2016-07-25 at 11 57 14 pm](https://cloud.githubusercontent.com/assets/4974609/17125674/8817f724-52c3-11e6-849b-f5b97f349733.png)

Removing this package will allow Meteor to start. Any ideas?

**Comment by juliancwirko** [comment]
Hi, hard to say, I need to investigate what might be the case. Could you test it on the same app but with Meteor 1.3 ? I wonder if this is something related to the new Meteor version. I'll try to test it on 1.4 asap.

**Comment by joncursi** [comment]
It works on Meteor 1.3.x, I had this package previously installed before attempting the Meteor 1.4 upgrade and that's where things broke.

**Comment by juliancwirko** [comment]
ok then, I need some time to do some research and testing. I'll keep you posted.

**Comment by juliancwirko** [comment]
Hi, it seems to work, sort of, but it needs loooooooooooooooooooong time to compile those styles ;) I've left it for about 15-20 minutes and it works like it should ;) Of course this can't be accepted, but it could be very hard to debug. At least for me :/

Maybe someone want to help with this? 

Sorry to say this, but in my opinion Meteor is useless when you want to use pro front-end tooling. Such tasks like PostCSS, CSS Modules configuration etc. should be as simple as adding one package/loader/plugin, and it is as simple as that with for example Webpack. You could try to use [webpack:webpack](https://atmospherejs.com/webpack/webpack) package. Or you could split your code like I've done [here](https://github.com/juliancwirko/react-redux-webpack-meteor)
For now I can't see a proper way to implement PostCSS in Meteor :/ 

You can also try with [nathantreid:css-modules](https://atmospherejs.com/nathantreid/css-modules) it has PostCSS support too, maybe it will work with Meteor 1.4

**Comment by PEM--** [comment]
Sad news 😞

**Comment by clayne11** [comment]
I'm using `nathantreid:css-modules` and the situation isn't any better. Hopefully the Meteor team can resolve this quickly. I obviously can't update to `1.4` until this is fixed.

**Comment by juliancwirko** [comment]
Oh, so if this is something wrong with the build tool then this is a good news (sort of ;)).
There is a hope that it will work without another amount of weird hacks in this package (just to be able to use PostCSS in Meteor) ;)

**Comment by joncursi** [comment]
@clayne11 did you by chance create an issue on the Meteor github repo? I only created one here because my initial thought was that this was a package issue not a meteor issue, but if multiple packages have the same issue, maybe we should surface this as an issue on the meteor repo as well

EDIT: looks like @crazile already did this! thanks!

**Comment by clayne11** [comment]
Yes I did create an issue.

https://github.com/meteor/meteor/issues/7493

**Comment by namirsab** [comment]
I'm only using the auto-prefixer plugin and it works.
I have to check which plugin is delaying the processt that much

**Comment by clayne11** [comment]
The situation has improved immensely with the Meteor 1.4.2 betas.

**Comment by thegillis** [comment]
In case this helps anyone else, I was able to track this down to a CSS import with the `postcss-easy-import` plugin. I had found an example import for my main css file similar to:

```
@import 'react-treeview/react-treeview.css';
```

where `react-treeview` was in my `node_modules` folder. Build times were improved by an order of magnitude when I changed my import to:

```
@import '../node_modules/react-treeview/react-treeview.css';
```

I have not had the chance yet to track down the exact cause yet.
